packages/core/src/components/Accordion.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import {
+  View,
+  StyleSheet,
+  StyleProp,
+  TextStyle,
+  Pressable,
+} from "react-native";
+import Text from "./Text";
+import { withTheme } from "../theming";
+import type { IconSlot } from "../interfaces/Icon";
+import type { Theme } from "../styles/DefaultTheme";
+import { colorTypes } from "@draftbit/types";
+
+import {
+  COMPONENT_TYPES,
+  createTextProp,
+  createNumberProp,
+  createIconProp,
+  createColorProp,
+} from "@draftbit/types";
+
+type Props = {
+  openColor: colorTypes;
+  closedCloser: colorTypes;
+  caretColor: colorTypes;
+  icon?: string;
+  iconSize: number;
+  style: StyleProp<TextStyle>;
+  children: React.ReactNode;
+  title: string;
+  theme: Theme;
+} & IconSlot;
+
+const Accordion = ({
+  Icon,
+  openColor,
+  closedCloser,
+  //caretColor,
+  icon,
+  iconSize = 24,
+  style,
+  title,
+  children,
+  theme: { colors },
+}: Props) => {
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handlePressAction = () => {
+    setExpanded(!expanded);
+  };
+
+  const titleColor = expanded ? colors[openColor] : colors[closedCloser];
```

Comment:
the theme color that's being passed in is the color itself, so `colors[openColor]` won't necessarily work.

You can do something like this though:
```
const expandedColor = openColor || theme.colors.primary
const collapsedColor = closedColor || theme.colors.divider

const titleColor = expanded ? expandedColor : collapsedColor
```

Reply:
done